### PR TITLE
feat: creating new badge feature

### PR DESCRIPTION
**Pull request by mariyasada**
done with badge feature

### CODE REVIEW

**Review by harshitadaswani — COMMENTED** [review]
Just a suggestion to indent your code properly with spaces for better readability of your code, otherwise, you are good to go!

File: htmlFiles/avatar.html [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+<!-- ---------------------------------------------------------javascript functionality------------------------------- -->
+ <script>
+                const avatarCodeBtn = document.querySelector("#avatar-code-btn");
+                const avatarDiv = document.querySelector("#avatar-div-code");
+
+                avatarCodeBtn.addEventListener("click",() =>{
+                        console.log("clicked");
+                        avatarDiv.style.display =="block" ? avatarDiv.style.display ="none": avatarDiv.style.display ="block";
+                    });
+
```
Comment (naufilshaikh09) [inline htmlFiles/avatar.html]:
```suggestion
Spacing should be consistent for the all files.
```

**Review by naufilshaikh09 — COMMENTED** [review]
Spacing should be consistent for the all files.

**Review by naufilshaikh09 — COMMENTED** [review]
Spacing should be consistent for the all files.

**Pull request merged**